src/main/java/edu/greenblitz/robotName/commands/getNoteToSystem/CollectNoteToScoringMode.java

```diff
@@ -16,7 +14,7 @@ public CollectNoteToScoringMode() {
                 new ConditionalCommand(
                         new TransferNote(),
```

Comment:
Conditional of transfer note and empty command?

Reply:
If I want to bring the note to the arm I transfer else I do nothing

Reply:
Hevanti, 10x